backend/fault.py

```diff
@@ -80,10 +91,18 @@ class AlarmSeverity(DefaultIntEnum):
             raise PVInvalidError(self.pv.pvname)
 
         if self.ok_value is not None:
+            # "is not None" means it has a value. It is not a blank, no value
+            # ok_value = value stated in spreadsheet
+            # obj.value = actual reading value from pv
             return obj.val != self.ok_value
+            # Does the actual value NOT match the spreadsheet value?
+            # If they don't match, return TRUE
+            # return "FALSE" means is_okay, not faulted
 
         elif self.fault_value is not None:
             return obj.val == self.fault_value
+            # return "FALSE" means not faulted
```

Comment:
This is just a stylistic thing for me, but I prefer comments above the line they're referencing

Reply:
Wow, I have no memory of any of these comments but I do like them!

Reply:
Genuinely can't remember if you added these to help me, or if I added these while trying to relearn the code lol

Reply:
I wouldn't be surprised if I was live typing these to explain what I was doing lmfaooo